rust/error/src/lib.rs

```diff
@@ -39,6 +39,8 @@ pub enum ErrorCodes {
     Unavailable = 14,
     // DATA_LOSS indicates unrecoverable data loss or corruption.
     DataLoss = 15,
+    // VERSION_MISMATCH indicates a version mismatch.
```

Comment:
can we add a comment that this is a chroma specific error code, not a GRPC one - since the comment at the top indicates that we mirror grpc?

Reply:
Yes.